### PR TITLE
www.cumhuriyet.com.tr

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/119586

### CODE REVIEW

File: SpywareFilter/sections/tracking_servers.txt [hunk 1]
```diff
@@ -54,6 +54,7 @@
 !
 ||reports.koalametrics.com^
 ||js.go2sdk.com^
+||srvr.dmvs-apac.com^
```
Comment (dcxstq2) [inline SpywareFilter/sections/tracking_servers.txt:57]: It also blocks embedded Dailymotion videos on some websites.

<details>
 <summary>Screenshot</summary>

resim


</details>

```
https://www.preview.ph/
https://www.esquiremag.ph/culture/movies-and-tv/stranger-things-season-4-volume-1-review-a2384-20220527-src-esquire?ref=home_feed_1
https://www.smartparenting.com.ph/
https://www.femalenetwork.com/
```

Comment (dcxstq2) [reply SpywareFilter/sections/tracking_servers.txt:57]: And other than that, can you also add the following rules for right/left ad spaces?

```
cumhuriyet.com.tr##.page-left-fixed-ad
cumhuriyet.com.tr##.page-right-fixed-ad
```

Comment (zloyden) [reply SpywareFilter/sections/tracking_servers.txt:57]: I don't think that we should block it.
However, this works fine on my end.
```
||static1.dmcdn.net/playerv5/dmp.pes_pip_tracking.*.js
```

[For this](https://github.com/AdguardTeam/AdguardFilters/pull/119956/files#r883357299), one rule can be enough
```
cumhuriyet.com.tr##div[class^="page-"][class*="fixed-ad"]
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/specific.txt [hunk 2]
```diff
@@ -54,6 +54,7 @@
 !
 ||androidpolice.com/pixel.png?
 ||kiwisizing.com/api/log
+||static1.dmcdn.net/playerv5/dmp.pes_pip_tracking.*.js
```
Comment (zloyden) [inline SpywareFilter/sections/specific.txt:57]: I've added this directly, please remove it from the pull request

**Review by zloyden — APPROVED** [review]

**Pull request merged**